tests/suites/test_suite_rsa.function

```diff
@@ -922,11 +925,24 @@ void mbedtls_rsa_private( data_t * message_str, int mod, int radix_P,
         TEST_ASSERT( hexcmp( output, result_hex_str->x, ctx2.len, result_hex_str->len ) == 0 );
     }
 
+    /* And now one more time only with N and D */
+    TEST_ASSERT( mbedtls_rsa_export( &ctx2, &N, &P, &Q, &D, &E ) == 0 );
+    TEST_ASSERT( mbedtls_rsa_import( &ctx3, &N, NULL, NULL, &D, NULL ) == 0 );
```

Comment:
The documentation of `mbedtls_rsa_import` states that you must call `mbedtls_rsa_complete` afterwards. I disagree with lifting this restriction.

C being C, it isn't currently enforced at the language level. But we don't guarantee that it will keep working in the next version of Mbed TLS. And please note that the next major version (3.0) is likely to remove these low-level functions from the public interface.